packages/devtools-reps/src/reps/grip.js

```diff
@@ -78,6 +78,10 @@ function GripRep(props) {
 
   const propsArray = safePropIterator(props, object, maxLengthMap.get(mode));
 
+  if (mode === MODE.EXPANDED) {
```

Comment:
could we move this before building `propsArray` ? `safePropIterator` can be a bit costly, and here we don't use it when in expanded mode.

Reply:
Right observation! I think it's possible, now I move it.